### DISCUSSION TITLE
Konfiguration der Cronjobs per Weboberfläche

### DISCUSSION THREAD

**Discussion by RienSte** [wiggal/GEN24_Ladesteuerung#89] [Ideas]
Hi,

ist es möglich, die Konfiguration der Cronjobs auch in die Weboberfläche aufzunehmen?
Also beispielsweise welche Wetterdienste man verwenden möchte, falls mehrere konfiguriert sind?

Habe aktuell das Thema, dass dein Tool bei Bekannten auch im Einsatz ist, die weniger IT-affin sind. D.h. hier bin ich direkt zum Sysadmin nominiert worden :-) wenn nun aber was in der grundlegenden Config (bspw. Cronjobs oder git pull) gemacht werden muss, muss ich immer hinfahren, teilweise mit Monitor im Gepäck (weil nicht vor Ort) usw.

Wär schön, wenn man sowas auch per Telefonsupport machen könnte.

Danke und SG
Stephan

**Comment by wiggal** [comment]
theoretisch ist es möglich, wird aber wenn überhaupt sehr lange dauern (heuer nicht mehr), es sind noch viele Punkte in der Warteschleife.
Nur so als Hinweis, ich nehm keine Adminaufgaben ohne VPN-Zugang an.

**Comment by wiggal** [comment]
Sieh dir mal dieses Tool an, vielleicht ist es was du suchst.
https://github.com/fisherinnovation/CronWeb
